tests/contrib/elasticsearch/test.py

```diff
@@ -78,10 +80,23 @@ def test_elasticsearch(self):
         eq_(span.get_tag(metadata.URL), "/%s/%s/%s" % (self.ES_INDEX, self.ES_TYPE, 10))
         eq_(span.resource, "PUT /%s/%s/?" % (self.ES_INDEX, self.ES_TYPE))
 
+        # Make the data available
+        es.indices.refresh(index=self.ES_INDEX)
+
+        spans = writer.pop()
+        assert spans, spans
+        eq_(len(spans), 1)
+        span = spans[0]
+        eq_(span.resource, "POST /%s/_refresh" % self.ES_INDEX)
+        eq_(span.get_tag(metadata.METHOD), "POST")
+        eq_(span.get_tag(metadata.URL), "/%s/_refresh" % self.ES_INDEX)
```

Comment:
The search wasn't actually returning any results because a refresh wasn't called. I copied this from the patch test below.